django/shortcuts.py

```diff
@@ -114,6 +122,11 @@ def get_list_or_404(klass, *args, **kwargs):
     return obj_list
 
 
+async def aget_list_or_404(klass, *args, **kwargs):
+    """See get_list_or_404()."""
+    return await sync_to_async(get_list_or_404)(klass, *args, **kwargs)
```

Comment:
`QuerySet.aget()` and `afilter()` are already implemented. We should provide an async variants not only wrappers on existing functions.

Reply:
let's go for it !

Reply:
Hi @felixxm 

there's no `filter()` for a [very good reason ](https://code.djangoproject.com/ticket/34147). I get the idea.